src/number/n_GF.jl

```diff
@@ -337,24 +337,12 @@ promote_rule(C::Type{n_GF}, ::Type{n_Z}) = n_GF
 #
 ###############################################################################
 
-(R::N_GField)() = n_GF(R)
-
-(R::N_GField)(x::Integer) = R(libSingular.n_InitMPZ(BigInt(x), R.ptr))
-
-(R::N_GField)(n::Int) = n_GF(R, n)
-
-# not currently implemented
-# function (R::N_GField)(n::n_Z)
-#    m = libSingular.nApplyMapFunc(R.from_n_Z, n.ptr, parent(n).ptr, R.ptr)
-#    return n_GF(R, m)
-# end
```

Comment:
OK, I see now that this was commented out because it triggers a segfault. That could be a bug in Singular -- @hannes14 can you have a look at this? We are using the map `R.from_n_Z` which was created using `libSingular.n_SetMap(ZZ.ptr, ptr)`.

Reply:
Hans fixed this in Singular master; but I also added a workaround here in this PR.

So I think this could be merged now.